proposals/description/fip_76b.ts

```diff
@@ -0,0 +1,106 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_76b: ProposalDescription = {
+  title: 'FIP-76b: Add LUSD to FeiRari',
+  commands: [
+    {
+      target: 'delayedPCVMoverWethUniToBal',
+      values: '0',
+      method: 'unpause()',
+      arguments: [],
+      description: 'Unpause FEI/WETH delayed PCVMover'
+    },
+    {
+      target: 'delayedPCVMoverWethUniToBal',
+      values: '0',
+      method: 'withdrawRatio()',
+      arguments: [],
+      description: 'Finish migration of FEI/WETH to Balancer'
+    },
+    {
+      target: 'balancerDepositFeiWeth',
+      values: '0',
+      method: 'deposit()',
+      arguments: [],
+      description: 'Deposit WETH in Balancer'
+    },
+    {
+      target: 'aaveEthPCVDeposit',
+      values: '0',
+      method: 'withdraw(address,uint256)',
+      arguments: ['{uniswapPCVDeposit}', '400000000000000000000'],
+      description: 'Send 400 WETH to Uniswap PCVDeposit'
+    },
+    {
+      target: 'uniswapPCVDeposit',
+      values: '0',
+      method: 'deposit()',
```

Comment:
this call could fail if the pool is out of bounds relative to chainlink, do you think we should increase slippage tolerance?

Reply:
there are still 40M$ of liquidity in the meantime, should be ok?

Reply:
Chainlink could be off irrespective of the liquidity amount if ETH is globally a different price on exchanges than on mainnet DeFi. Its infrequent so this is fine